lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -9,6 +9,7 @@
 /* eslint-disable no-console */
 
 const fs = require('fs');
+const mkdirp = require('mkdirp');
```

Comment:
we really should have switched everything over to `make-dir` (or `fs.mkdir` after the move to node 10), but good for now